tensorflow/core/graph/gpu_fusion_pass.cc

```diff
@@ -394,6 +423,7 @@ bool ROCmFusionOpConvolutionBiasBatchNormActivation::IsFusionEligible(
           d->bias = n2;
           d->norm = n3;
           d->actv = n4;
+          VLOG(kVlogLevel) << "===========";
```

Comment:
remove this?

Reply:
thats by design to make looking at the VLOG output easier on the eye...would like to leave it in

Reply:
yeah I looked at your other codes down below and it appears to be that way. ok to leave it as-is.